src/client/extension.ts

```diff
@@ -270,6 +263,47 @@ export async function activate(
         vscode.commands.executeCommand('setContext', 'powerpages.websiteYmlExists', false);
     }
 
+    const registerPreviewShowCommand = async () => {
```

Comment:
Can we move the preview related code to a separate file?

Reply:
this is registering command so extension.js is right location for it, moved the other code of preview site to separate file